.github/workflows/test-suite.yml

```diff
@@ -32,29 +35,14 @@ jobs:
             zsh -n "$file"
             echo "Success! 🎉"
           done
-      - name: Install Test - No Branch - Master
+      - name: Install Test - branch "${{ matrix.branch }}"
         shell: zsh {0}
         run: |
-          # Ensure Zap not already installed
-          rm -rf "${XDG_DATA_HOME:-$HOME/.local/share}/zap"
           # Init
-          zsh ${{ github.workspace }}/install.zsh
+          zsh ${{ github.workspace }}/install.zsh --branch ${{ matrix.branch }}
           source "${XDG_DATA_HOME:-$HOME/.local/share}/zap/zap.zsh"
           # Test Environment Variable
           [[ -d "$ZAP_DIR" ]]
           # Test Plugin Installation
-          plug "wintermi/zsh-lsd"
-          git -C "${ZAP_DIR}/plugins/zsh-lsd/" status || exit 1
-      - name: Install Test - Branch "release-v1"
-        shell: zsh {0}
-        run: |
-          # Ensure Zap not already installed
-          rm -rf "${XDG_DATA_HOME:-$HOME/.local/share}/zap"
-          # Init
-          zsh ${{ github.workspace }}/install.zsh --branch release-v1
-          source "${XDG_DATA_HOME:-$HOME/.local/share}/zap/zap.zsh"
-          # Test Environment Variable
-          [[ -d "$ZAP_DIR" ]]
-          # Test Plugin Installation
-          plug "wintermi/zsh-lsd"
-          git -C "${ZAP_DIR}/plugins/zsh-lsd/" status || exit 1
+          plug "zsh-users/zsh-syntax-highlighting"
```

Comment:
This is already installed by default.

This needs to be a plugin that is not present in the `default-zshrc` file.